src/arcade/core/util/exceptions/IncorrectlySetParameterException.java

```diff
@@ -0,0 +1,14 @@
+package arcade.core.util.exceptions;
+
+/** Exception thrown when parameter value is not equal to single expected value. */
+public class IncorrectlySetParameterException extends RuntimeException {
```

Comment:
picking nits, but IncorrectlySet feels like a developer error rather than an user error (perhaps the point?) 
```suggestion
public class InvalidParameterValueException extends RuntimeException {
```

Reply:
I agree I don't love IncorrectlySet but Invalid I think makes it sounds like the parameter is of the wrong type as opposed to the wrong specific value (I started with the name InvalidParameter and chanced it to try to address this, I don't mind changing it back but wonder if there is yet another option that would best capture the intent here)

Reply:
Does `InvalidParameterValue` not imply the value?

Reply:
ah my eyes skipped the "Value" part